src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,87 @@
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
+      <tbody>
+        {visibleTodos.map(todo => (
+          <tr
+            data-cy="todo"
+            className={cn('', {
+              'has-background-info-light': selectedTodo?.id === todo.id,
+            })}
+            key={todo.id}
+          >
+            <td className="is-vcentered">{todo.id}</td>
 
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">6</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">
-            qui ullam ratione quibusdam voluptatem quia omnis
-          </p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
+            <td className="is-vcentered">
+              {todo.completed && (
+                <span className="icon" data-cy="iconCompleted">
+                  <i className="fas fa-check" />
+                </span>
+              )}
+            </td>
 
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">8</td>
-        <td className="is-vcentered">
-          <span className="icon" data-cy="iconCompleted">
-            <i className="fas fa-check" />
-          </span>
-        </td>
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-success">quo adipisci enim quam ut ab</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-    </tbody>
-  </table>
-);
+            <td className="is-vcentered is-expanded">
+              <p
+                className={cn('has-text', {
+                  'has-text-danger': !todo.completed,
+                  'has-text-success': todo.completed,
+                })}
+              >
+                {todo.title}
+              </p>
+            </td>
+            <td className="has-text-right is-vcentered">
+              <button data-cy="selectButton" className="button" type="button">
+                {selectedTodo?.id === todo.id ? (
+                  <span className="icon">
+                    <i
+                      className="far fa-eye-slash"
+                      onClick={() => handleSelectPost(todo)}
+                    />
+                  </span>
+                ) : (
+                  <span className="icon">
+                    <i
+                      className="far fa-eye"
+                      onClick={() => handleSelectPost(todo)}
```

Comment:
The `onClick` handler should be moved from the `i` element to the `button` element. This ensures that the button is the interactive element, improving accessibility and making the click area larger for better user experience.